Sensors.cpp

```diff
@@ -356,13 +294,71 @@ bool Sensors::pmSensorRead() {
     }
 }
 
+void Sensors::H&TRead() {
```

Comment:
I liked this implementation, but:

- The H&T type should be a parameter on the main public method sensors.init() but
- Also we need a dynamic library, we can't have **define** any, the users will be define that they need.
- A think so that we can try to load all init methods, and this libraries can try to define if each object is null or not, for this reason we dont need this case, we only need try to load all sensors (library objects)

Also, we need a example on examples directory that tests this methods.